napari/layers/utils/plane_manager.py

```diff
@@ -77,3 +73,59 @@ def from_points(cls, a, b, c):
         plane_normal = np.cross(ab, ac)
         plane_position = np.mean(abc, axis=0)
         return cls(position=plane_position, normal=plane_normal)
+
+
+class PlaneList(EventedList):
```

Comment:
it might be nice to create a convenience function/method for creating a `PlaneList` for an axis aligned bounding box (6 planes) centered around a position in a volume with certain dimensions in 0,1,2, obviously this isn't necessary for this PR

Reply:
Totally! I was planning to do that already in a follow up PR, but first I want to be sure on how these models should look like here.

Reply:
I decided to do this, since it wasn't too hard. Try this on the example:
```python
volume_layer.clipping_planes.from_bounding_box([32, 32, 32], [40, 20, 30]) # (center, dimensions)
```